### ISSUE TITLE
Wrong path to package.json for commonjs

### ISSUE THREAD

**Issue by jeremybarbet** [segmentio/analytics-react-native#572]
- `analytics-react-native` version: 2.2.2
- Integrations versions (if used):
- React Native version: 0.67.4
- iOS or Android or both? both

**Steps to reproduce**

- Follow this step https://github.com/segmentio/analytics-react-native#setting-up-the-client
- Running the app results in a module not found error `../package.json in context.js`

**Expected behavior**

- To find the `package.json` file

**Actual behavior**

After bundled, `context.js` is in `lib/commonjs/context.js` rather than in `src/context.ts`. The path to the `package.json` is then wrong

That's the patch I'm using until the path can be determined based on the root directory:

```patch
diff --git a/node_modules/@segment/analytics-react-native/lib/commonjs/context.js b/node_modules/@segment/analytics-react-native/lib/commonjs/context.js
index 19e2207..b8fbea4 100644
--- a/node_modules/@segment/analytics-react-native/lib/commonjs/context.js
+++ b/node_modules/@segment/analytics-react-native/lib/commonjs/context.js
@@ -7,7 +7,7 @@ exports.getContext = void 0;
 
 var _reactNative = require("react-native");
 
-var _package = _interopRequireDefault(require("../package.json"));
+var _package = _interopRequireDefault(require("../../package.json"));
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
```

**Comment by jeffcongdon** [comment]
experiencing this same behavior outside of commonjs, running test suite in jest.  Using version `2.2.2`, but confirmed error goes back to `2.0.0-pilot`

**Comment by oscb** [comment]
This should be fixed in `2.4.0`
